js/index.js

```diff
@@ -81,10 +84,13 @@ var View = {
   init: function() {
     var source = $('#board-template').html();
     this.template = Handlebars.compile(source);
+
+    console.log('it works here');
```

Comment:
You may want to give yourself some more context in these console statement. For instance here, you might say
```js
console.log('it works here in the View init function');
```